src/components/Catalogs/Catalogs.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+import classes from "./Catalogs.module.css";
+import icon from "../../images/placeholder.jpg";
+
+
+const Catalog = (props) => {
+  // Initial state for tracking catalogs
+  const [catalogsState, setCatalogsState] = useState({
+    providers: null,
+    isLoading: false,
+    error: null,
+  });
+
+  let resultLength = 0;
+
+  const catalogs_api_url = process.env.REACT_APP_XIS_CATALOGS_API;
+  let result = (
+    <div>
+      Error loading catalogs. Please contact an administrator.
```

Comment:
Nit: "Please contact your administrator".  Please update all instances of this message to match.